pycqed/tests/test_pulsar_element.py

```diff
@@ -90,63 +90,63 @@ def test_fixpoint(self):
 
-    def test_operation_dependent_buffers_and_compensation(self):
-        # Fixed point should shift both elements by 2 ns
-
-        RO_amp = 0.1
-        MW_amp = 0.3
-        Flux_amp = 0.5
-        operation_dict = {'RO q0': {'amplitude': RO_amp,
-                                    'length': 300e-9,
-                                    'operation_type': 'RO',
-                                    'channel': 'ch1',
-                                    'pulse_delay': 0,
-                                    'pulse_type': 'SquarePulse'},
-                          'MW q0': {'amplitude': MW_amp,
-                                    'length': 20e-9,
-                                    'operation_type': 'MW',
-                                    'channel': 'ch1',
-                                    'pulse_delay': 0,
-                                    'pulse_type': 'SquarePulse'},
-                          'Flux q0': {'amplitude': Flux_amp,
-                                      'length': 40e-9,
-                                      'operation_type': 'Flux',
-                                      'channel': 'ch1',
-                                      'pulse_delay': 0,
-                                      'pulse_type': 'SquarePulse'},
-
-                          'sequencer_config': {'Buffer_Flux_Flux': 1e-9,
-                                               'Buffer_Flux_MW': 2e-9,
-                                               'Buffer_Flux_RO': 3e-9,
-                                               'Buffer_MW_Flux': 4e-9,
-                                               'Buffer_MW_MW': 5e-9,
-                                               'Buffer_MW_RO': 6e-9,
-                                               'Buffer_RO_Flux': 7e-9,
-                                               'Buffer_RO_MW': 8e-9,
-                                               'Buffer_RO_RO': 10e-9,
-                                               'RO_fixed_point': 1e-06,
-                                               'Flux_comp_dead_time': 3e-6}}
-        sequencer_config = operation_dict['sequencer_config']
-
-        fake_seq = ['MW q0', 'MW q0', 'Flux q0', 'MW q0', 'RO q0']
-        pulses = []
-        for p in fake_seq:
-            pulses += [operation_dict[p]]
-        test_elt = multi_pulse_elt(0, self.station, pulses, sequencer_config)
-
-        min_samples = 1800+3040  # 1us fixpoint, 300ns RO pulse and 4ns zeros
-        ch1_wf = test_elt.waveforms()[1]['ch1']
-        self.assertEqual(len(ch1_wf), min_samples)
-
-        expected_wf = np.zeros(min_samples)
-        expected_wf[1000:1300] = RO_amp
-        expected_wf[974:994] = MW_amp
-        expected_wf[932:972] = Flux_amp
-        expected_wf[908:928] = MW_amp
-        expected_wf[883:903] = MW_amp
-        expected_wf[4300:4340] = -Flux_amp
-
-        np.testing.assert_array_almost_equal(ch1_wf, expected_wf)
+    # def test_operation_dependent_buffers_and_compensation(self):
```

Comment:
Shouldn't this test be deprecated or skipped, instead of commented out?

Reply:
addressed!